src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -89,28 +88,40 @@ Copyright (c) .NET Foundation. All rights reserved.
           SkipUnchangedFiles=$(SkipUnchangedFiles);
           PreserveStoreLayout=$(PreserveStoreLayout);
           CreateProfilingSymbols=$(CreateProfilingSymbols);
-          StoreSymbolsStagingDir=$(StoreSymbolsStagingDir)
-        </Properties>
+          StoreSymbolsStagingDir=$(StoreSymbolsStagingDir);
+          DisableImplicitFrameworkReferences=false;
+        </AdditionalProperties>
       </PackageReferencesToStore>
     </ItemGroup>
 
+<!-- Restore phase -->
     <MSBuild Projects="@(PackageReferencesToStore)"
-                 Targets="StoreWorker"
+                 Targets="RestoreForComposeStore"
+                 Properties="EvaluationIteration=1;"
```

Comment:
What is `EvaluationIteration` here and below when set to `2` used for? I don't see these properties being consumed anywhere.